source/upgrade.txt

```diff
@@ -70,6 +70,10 @@ Version 6.x Breaking Changes
   ``tlsCertificateKeyFile`` connection options when you call the
   ``MongoClient.connect()`` method, not when you create the
   ``MongoClient`` instance.
+- The ``Db.command()`` method only accepts options that are not related
```

Comment:
Issue:

Perhaps my fault for providing this in the suggestion, but I think "only" is a misplaced modifier in this case.
From the [style guide on "only"](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#o):
> Place only preceding the word or group that it modifies. If only modifies a final word or phrase, it may precede or follow that word or phrase.

Suggestion:

I think In this case, "only" modifies options rather than "accepts".

```suggestion
- The ``Db.command()`` method accepts only options that are not related
```